client/trailseek/src/screens/ListTrailScreen.js

```diff
@@ -31,7 +31,7 @@ const ListTrailScreen = ({ route }) => {
       try {
         const results = await dispatch(fetchTrailsByQuery({ query, limit }))
         const uResults = unwrapResult(results)
-        setFilteredTrails(uResults)
+        setFilteredTrails(uResults.response)
```

Comment:
it returns a response in the redux end, so its supposed to be uResults... did you test this? Is it working?

Reply:
I fixed this bug with the search, it should be safe to merge now

Reply:
Merging